projects/react/package.json

```
@@ -0,0 +1,25 @@
+{
+    "name": "@maskito/react",
+    "version": "0.0.1",
```

Let's sync version with other Maskito's libraries:
```suggestion
    "version": "0.10.0",
```

---

done